.github/workflows/release.yaml

```diff
@@ -4,7 +4,13 @@ on:
   workflow_dispatch:
     inputs:
       tag:
-        description: 'The version to release (e.g. v1.2.3)'
+        description: |
+          The version to release.
+          Depending on the value, a production release will be published to GitHub or not.
+            - In case of prerelease tags (e.g. v1.2.3-alpha.1) it will build-push the images (only fullversion variants), 
```

Comment:
Do you mind elaborating what' s a "fullversion variant"?

Reply:
I borrowed this term from the [semver parsing action](https://github.com/booxmedialtd/ws-action-parse-semver#fullversion). It's a full representation of the semver, e.g. `v1.2.3-alpha.1`. It's used when preparing tags that the images are going to be tagged with in the `Add standard tags` and `Add Red Hat standard tags` steps of the `  build-push-images` job.

Reply:
I've changed the description to follow the convention we have for `standard` and `supplemental` tags in the `build-push-images` job: 22f0702